Web/Assignment-3/Assignment-3.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html>
+<head>
+<script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js"></script>
+<script type="text/javascript" src="jquery-1.11.3.js"></script>
```

Comment:
ACTION: This reference is not required here as you already using minified version of jquery file.